hazelcast/src/main/java/com/hazelcast/datastore/ExternalDataStoreFactory.java

```diff
@@ -41,6 +41,11 @@
      */
     void init(ExternalDataStoreConfig config);
 
+    /**
+     * Test data store connection.
+     */
```

Comment:
Missing `@since` tag,

It's not clear from the javadoc if you need to first init and the test, or you can just test, without init.